queue/test/fiskaltrust.Middleware.Storage.EF.AcceptanceTest/Fixtures/EfConnectionStringFixture.cs

```diff
@@ -1,11 +1,11 @@
-﻿using System;
+using System;
 
 namespace fiskaltrust.Middleware.Storage.EF.AcceptanceTest.Fixtures
 {
     public static class EfConnectionStringFixture
     {
-        public const string ServerConnectionString = @"Data Source=(LocalDb)\MSSQLLocalDB;Integrated Security=SSPI";
+        public const string ServerConnectionString = @"Data Source=(LocalDb)\MSSQLLocalDB;Integrated Security=SSPI;Connect Timeout=60;";
```

Comment:
Is this timeout only for the local installations or is this sth we should also recommend with current rollouts?

Reply:
We increased this because the instance we use in the acceptancetests is a bit undersized and slow. Normal world usage should be fine with the default timeout.